ormconfig.ts

```diff
@@ -0,0 +1,15 @@
+const ROOT_DIR = process.env.ROOT_DIR;
+
+export = {
+    entities: `${ROOT_DIR}/entities/*.ts`,
+    migrations: `${ROOT_DIR}/migrations/*.ts`,
+    subscribers: `${ROOT_DIR}/subscribers/*.ts`,
+    seedingFactories: `${ROOT_DIR}/services/db/factories/**/*{.ts}`,
+    seedingSeeds: `${ROOT_DIR}/services/db/seeders/**/*{.ts}`,
```

Comment:
all those should support both .ts and .js extensions (js is in production).

I'm not sure this file would end up in production image. Is it possible to set this in src somewhere?!